update/hll--2.10.sql

```diff
@@ -271,7 +271,7 @@ CREATE FUNCTION hll_set_output_version(integer)
 CREATE FUNCTION hll_set_max_sparse(integer)
      RETURNS integer
      AS 'MODULE_PATHNAME'
-     LANGUAGE C STRICT IMMUTABLE;
+  LANGUAGE C STRICT;
```

Comment:
Why we remove `immutable`?

Reply:
If it's immutable, then the optimizer will optimize it to constant in this statement: `select hll_set_max_sparse(0) from gp_dist_random('gp_id');`. But I modify the C code of this UDF later. So this modification is not needed anymore. I'll restore it.

Reply:
According to the `immutable` classification: `An IMMUTABLE function cannot modify the database and is guaranteed to return the same results given the same arguments forever.` It doesn't mention the different behavior of the QD and QE. I'm not sure either.